### ISSUE TITLE
High scores screen, the default name can be too long to fit the list

### ISSUE THREAD

**Issue by Branikolog** [ihhub/fheroes2#8778]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Windows

### Describe the bug

![image](https://github.com/ihhub/fheroes2/assets/55348946/62d15b66-0523-4273-97ef-85f7ed485354)

### Save file

None.

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,

Hey, I made some nice scores. :D

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu !
I was always confused, how did you manage to make them on my own PC. :)

**Comment by LeHerosInconnu** [comment]
@Branikolog,

> Hi, @LeHerosInconnu ! I was always wondering, how did you manage to make them on my own PC. :)

Ah, Ah, Ah,

In fact, the name "Unknown Hero" comes from the game Heroes of Might en Magic 2.
The first time I validated the window that lets you enter a name for the high score, I hit the "enter" key without typing a letter in the text field, just to see.
It worked without a letter, and the name "Unknown Hero" was then displayed in the high score list.
I thought it was funny, and I used it afterwards for everything to do with the Heroes of Might and Magic serie, not forgetting the French version "Le Héros Inconnu".
Now, every time I see that name in a screenshot I didn't take myself, it makes me smile. :)
